index.bs

```diff
@@ -0,0 +1,331 @@
+      </dd>
+    </dl>
+
+    <div class="note">
+      Note: However, depending on the implementation, it's still possible for
+      some media that exist on the same document to be allowed to autoplay when
+      the document {{Document/autoplayPolicy}} returns {{disallowed}}. It is
+      **recommended** for authors to check each media element’s autoplayPolicy
+      in order to get a complete result when document’s autoplayPolicy is
+      disallowed, if authors want to use that as a result to determine whether
+      media can be started playing.
+    </div>
+
+    <div class=example>
+      If a user-agent does not allow any media to autoplay by default, but
+      allows autoplay on those media elements which have been clicked by users
+      before. At first, both the document's {{Document/autoplayPolicy}} and the
+      media element's {{HTMLMediaElement/autoplayPolicy}} will be {{disallowed}}.
+
+      However, after a user clicks on a media element, then this media element's
+      {{HTMLMediaElement/autoplayPolicy}} might become {{allowed}} if a
+      user-agent decides to bless this element because that seems intended by
+      users. In this case, the document's {{Document/autoplayPolicy}} and other
+      media elements' {{HTMLMediaElement/autoplayPolicy}} (if any), which
+      haven't been clicked yet and are on the same document, will still be
+      {{disallowed}}.
+    </div>
+
+    <div class="note">
+      Note: If the document has child documents, then the result from child
+      documents could be different from the result of their parent documents
+      depending on the implementation.
+    </div>
+
+    <div class=example>
+      Assume that the top level document A in `foo.com` returns {{allowed}} and
+      it has an embedded iframe, which has another document B from `bar.com`. A
+      user-agent could either make child document B return the same result that
+      is inherited from the top level document A. Or make the document B return
+      a different result, eg. {{disallowed}}.
+
+      Doing the former helps to lower the complexity and make the behavior of
+      blocking autoplay more consistent. The latter helps providing a
+      finer-grained autoplay control.
+    </div>
+
+    <div class="note">
+      Note : In addition, document's {{Document/autoplayPolicy}} can also be
+      used to know whether [[webaudio inline]] can be allowed to start if a
+      user-agent blocks web audio by default.
+    </div>
+
+    <div class=example>
+      [Web audio](https://webaudio.github.io/web-audio-api/#allowed-to-start)
+      uses the [=sticky activation=] to determine if {{AudioContext}} can be
+      allowed to start. If the document is allowed to autoplay, then
+      {{AudioContext}} should also be allowed to start.
+
+      <pre class="lang-javascript">
+        var ac = new AudioContext;
+        if (document.autoplayPolicy == "allowed") {
```

Comment:
```suggestion
        if (document.autoplayPolicy === "allowed") {
```
 and same for the others? I think strict equality is generally preferred

Reply:
done, will update it later.